crud/common/schema.lua

```diff
@@ -133,22 +133,42 @@ local function get_space_schema_hash(space)
     return digest.murmur(msgpack.encode(space_info))
 end
 
+local function extract_fields(tuple, fields)
+    local extracted_fields = {}
+
+    for i, field in ipairs(fields) do
+        extracted_fields[i] = tuple[field]
+    end
+
+    return extracted_fields
+end
+
+local function get_partial_result(tuple, fields)
```

Comment:
What about `filter_result_fields`?.

I'm not sure that we need `extract_fields` as a separated function.

It's ok to use early-return at the function begin and do other things after that:

```lua
local function filter_result_fields(tuple, fields)
    if fields == nil or tuple == nil then
        return tuple
    end

   for i, field in ipairs(fields) do
      ...
   end

   return smth
end
```